Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
  * Allow qualified access to events from other contracts.
 
 Compiler Features:
+ * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output (i.e. ``--ir`` on the CLI or ``ir`` in Standard JSON). The order would depend on the C++ compiler used to build the Solidity compiler binary, and resulted in GCC builds generating slightly different (but equivalent) bytecode than Clang builds.
```

Comment:
We should try to keep changelog entries more concise.

Reply:
I intentionally made this longer because I wanted to mention the consequences of this bug. All chain explorers offering source verification are affected and need to apply a workaround so this should not be easily confused with some unimportant internal detail.

I tried to omit some detail while still keeping the important bits. Check how you like it now.